iceoryx_posh/source/mepoo/memory_manager.cpp

```diff
@@ -155,21 +191,25 @@ void MemoryManager::configureMemoryManager(const MePooConfig& f_mePooConfig,
     generateChunkManagementPool(f_managementAllocator);
 }
 
-SharedChunk MemoryManager::getChunk(const MaxSize_t f_size)
+SharedChunk MemoryManager::getChunk(const uint32_t payloadSize,
+                                    const uint32_t payloadAlignment,
+                                    const uint32_t customHeaderSize,
+                                    const uint32_t customHeaderAlignment) noexcept
 {
     void* chunk{nullptr};
     MemPool* memPoolPointer{nullptr};
-    uint32_t adjustedSize = MemoryManager::sizeWithChunkHeaderStruct(f_size);
-    uint32_t totalSizeOfAquiredChunk = 0;
+    uint32_t requiredChunkSize =
+        MemoryManager::requiredChunkSize(payloadSize, payloadAlignment, customHeaderSize, customHeaderAlignment);
+    uint32_t aquiredChunkSize = 0;
```

Comment:
Add U

Reply:
U know, every time one mentions the `U` a small little unicorn dies